pkg/storage/sqlite/schema/schema.sql

```diff
@@ -0,0 +1,44 @@
+CREATE TABLE IF NOT EXISTS "indexer" (
+    "id" INTEGER NOT NULL PRIMARY KEY AUTOINCREMENT,
+    "name" TEXT NOT NULL,
+    "priority" INTEGER NOT NULL DEFAULT 25,
+    "uri" TEXT NOT NULL,
+    "api_key" TEXT
+);
+
+CREATE TABLE IF NOT EXISTS "quality_definition" (
+    "id" INTEGER NOT NULL PRIMARY KEY AUTOINCREMENT,
+    "quality_id" INTEGER NOT NULL,
+    "name" TEXT NOT NULL,
+    "preferred_size" NUMERIC NOT NULL,
+    "min_size" NUMERIC NOT NULL,
+    "max_size" NUMERIC NOT NULL,
+    "media_type" TEXT NOT NULL
+);
+
+CREATE TABLE IF NOT EXISTS "quality_profile" (
+    "id" INTEGER NOT NULL PRIMARY KEY AUTOINCREMENT,
+    "name" TEXT NOT NULL,
+    "cutoff" INTEGER NOT NULL,
```

Comment:
This cutoff is referencing the quality_id of a quality_definition? Can we name it like `cutoff_quality_id` or something of that nature

Reply:
yes, changed